students/johnpharmd/lesson08/circle_class.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+
+class Circle:
+    """represents a simple circle, using properties, a class method, and
+    special methods"""
+
+    @classmethod
+    def sort(cls, radius_list):
+        return sorted(radius_list)
+
+    def __init__(self, radius):
+        self.radius = radius
+
+    def __str__(self):
+        return 'Circle with radius: ' + str(self.radius)
+
+    def __repr__(self):
+        return 'Circle(' + str(self.radius) + ')'
+
+    def __add__(self, other):
+        self.radius += other.radius
+        return self.__repr__()
```

Comment:
These methods should return new circle objects, rather than changing the radius value of the circle that calls them